components/admin/requests/forms/PermitHolderInformationForm.tsx

```diff
@@ -36,26 +67,40 @@ export default function PermitHolderInformationForm({
           <FormControl isRequired>
             <FormLabel>{'First name'}</FormLabel>
             <Input
-              value={permitHolderInformation.firstName}
-              onChange={event =>
-                onChange({
-                  ...permitHolderInformation,
-                  firstName: event.target.value,
-                })
-              }
+              value={props.permitHolderInformation.firstName}
+              onChange={event => {
+                if (props.type === 'replacement') {
+                  props.onChange({
+                    ...props.permitHolderInformation,
+                    firstName: event.target.value,
+                  });
+                } else {
+                  props.onChange({
+                    ...props.permitHolderInformation,
+                    firstName: event.target.value,
+                  });
+                }
```

Comment:
Are these conditionals necessary?

Also to reduce the amount of duplicated code (doing object destructuring for every form field), we can make a higher-order function as discussed in this thread: https://github.com/uwblueprint/richmond-centre-for-disability/pull/112#discussion_r773466014

Reply:
Yep the conditions are necessary, even though the if and else cases are the same

Reply:
Hm interesting, we will look into this in the future then